exporter/stackdriver/stats.go

```diff
@@ -263,10 +263,14 @@ func (e *statsExporter) createMeasure(ctx context.Context, vd *view.Data) error
 
 	var metricKind metricpb.MetricDescriptor_MetricKind
 	var valueType metricpb.MetricDescriptor_ValueType
+	unit := m.Unit()
 
 	switch agg.Type {
 	case view.AggTypeCount:
 		valueType = metricpb.MetricDescriptor_INT64
+		// If the aggregation type is count, which counts the number of recorded measurements, the unit must be "1",
+		// because this view does not apply to the recorded values.
+		unit = "1"
```

Comment:
use `stats.UnitNone`, which is the corresponding const.

Reply:
done.